docs/radar.js

```diff
@@ -437,6 +453,10 @@ function radar_visualization(config) {
       blip.append("path")
         .attr("d", "M -11,-5 11,-5 0,13 z") // triangle pointing down
         .style("fill", d.color);
+    } else if (d.moved == null) {
```

Comment:
Unfortunately, making `moved` nullable is not backwards-compatible as generation scripts may assume that the value is non-nullable. Also, the README shows the full semantics of the field reinforcing this expectation. Internally, to compute the public tech radar visualization, we use a script that calculates the `moved` attribute based on the ring index.

The current radar code does not strictly enforce the values `1` (up) or `-1` (down) for `moved` and just checks the sign. If we consider entries with `moved = 2` as "new" (★), we'd need to adjust the two conditions in the blip shape computation.

Could you please adjust the code in lines 447-479 to enforce:
- moved = 2 - new (★)
- moved = 1 - up (▲)
- moved = 0 - no-change (〇)
- moved = -1 - down (▼)
?

Thanks 🙏